readme.md

```diff
@@ -86,10 +86,10 @@
 ## Programming Languages
 
 - [JavaScript](https://github.com/sorrycc/awesome-javascript)
-	- [Promises](https://github.com/wbinnssmith/awesome-promises)
+	- [Promises](https://github.com/wbinnssmith/awesome-promises) - Resources for JavaScript Promises.
```

Comment:
@legodude17 I don't think that meets the requirements outlined in https://github.com/sindresorhus/awesome/issues/928. At some point it stops being "new".